docs/fundamentals/networking/quic/quic-overview.md

```diff
@@ -161,8 +161,8 @@ var serverConnectionOptions = new QuicServerConnectionOptions
     // Same options as for server side SslStream.
     ServerAuthenticationOptions = new SslServerAuthenticationOptions
     {
-        // List of supported application protocols, must be the same or subset of QuicListenerOptions.ApplicationProtocols.
-        ApplicationProtocols = new List<SslApplicationProtocol>() { "protocol-name" },
+        // Specify the application protocols that the server supports. This list must be a subset of the protocols specified in QuicListenerOptions.ApplicationProtocols.
+        ApplicationProtocols = new List<SslApplicationProtocol>() { new SslApplicationProtocol("protocol-name") },
```

Comment:
Collection expressions will indeed simplify the code, but this will obscure the type and structure of `ApplicationProtocols` potentially making the sample code less clear to those unfamiliar with it.

How about this?

```suggestion
        ApplicationProtocols = new List<SslApplicationProtocol> { new("protocol-name") },
```

Or this?

```suggestion
        ApplicationProtocols = [new SslApplicationProtocol("protocol-name")],
```

Reply:
> Collection expressions will indeed simplify the code, but this will obscure the type and structure of ApplicationProtocols potentially making the sample code less clear to those unfamiliar with it.

Yes, you're right. Thanks for bringing this up.

IMO, second one looks better, but it's totally up to you.

Reply:
Thanks! I also prefer the second one.